src/connection/mqtt_client.c

```diff
@@ -968,6 +968,26 @@ int neu_mqtt_client_set_id(neu_mqtt_client_t *client, const char *id)
     return 0;
 }
 
+int neu_mqtt_client_set_will_msg(neu_mqtt_client_t *client, const char *topic,
```

Comment:
add another method to clean will msg ?

Reply:
I haven't found the correct way to clear the will message on the client side, perhaps this isn't the regular way to use it, any suggestions?